docker-build/run.sh

```diff
@@ -28,4 +28,5 @@ docker run \
   --publish=9090:9090 \
   --cap-add=sys_nice \
   ${GPU_FLAGS:+--gpus=${GPU_FLAGS}} \
-  "$INVOKEAI_TAG" ${1:+$@}
+  "$INVOKEAI_TAG" ${1:+$@} \
+  $CMD_OVERRIDE
```

Comment:
It is already working to pass Parameters to run.sh which will then overwrite the Container's default CMD.

Reply:
Got it, thanks! I was concerned about the same thing, so this is great. Is my writeup in the PR the correct way to do it, eg: './docker-build/run.sh --cors https://localhost:9090,http://10.42.0.20:8080`?

Reply:
To pass those parameters to invoke.py command: yes
If `--cors` accepts a comma separated list: IdK 🙈